dashboard/app/views/levels/editors/_md_instructions.haml

```diff
@@ -1,4 +1,7 @@
 .field
+  = f.label :instructions
+  = f.text_area :instructions, placeholder: 'Instructions', rows: 4
+.field
   = f.label :markdown_instructions
   %p When specified, these instructions will render in the instructions popup.
   %p The regular instruction field above will still display as the instructions below the game area.
```

Comment:
Is this true in the CSP usages? If not, could now preface this line with e.g. "In Blockly/Droplet levels,"